Helper/Data.php

```diff
@@ -86,15 +93,51 @@ public function getReachConfig($code, $storeId = null)
         return $this->getConfigValue(self::XML_PATH_REACH .'reach_gointerpay/'. $code, $storeId);
     }
 
-     /**
-     * Check Reach Enabled
+    /**
+     * Get Currency Options value REACH_CURRENCY_SWITCH - Function 06
```

Comment:
out of curiosity, what's up with the Function numbers?

Reply:
I'll do another update and remove that. I numbered them and then checked them off as I went through and tested each function. Wanted to make sure that I did a complete cart and purchase while watching those checkboxes.